discovery/eureka/eureka.go

```diff
@@ -157,10 +157,14 @@ func targetsForApp(app *Application) []model.LabelSet {
 
 	// Gather info about the app's 'instances'. Each instance is considered a task.
 	for _, t := range app.Instances {
-		targetAddress := net.JoinHostPort(t.HostName, strconv.Itoa(t.Port.Port))
+		var targetAddress string
+		if t.Port != nil {
```

Comment:
should we then use 80 as port? or not add the port to the address? @brian-brazil

Reply:
There has to be an address label. This hasn't come up before, I'd suggest just hardcoding to 80 for now.